dist/index.js

```diff
@@ -13533,11 +13533,11 @@ const toolCache = __importStar(__nccwpck_require__(7784));
 const core = __importStar(__nccwpck_require__(2186));
 const exec = __importStar(__nccwpck_require__(1514));
 // Define constants
-const GRUNTWORK_GITHUB_ORG = "gruntwork-io";
-const PATCHER_GITHUB_REPO = "patcher-cli";
-const PATCHER_VERSION = core.getInput("patcher_version") || "v0.15.1";
-const TERRAPATCH_GITHUB_REPO = "terrapatch-cli";
-const TERRAPATCH_VERSION = "v0.1.6";
+const GRUNTWORK_GITHUB_ORG = core.getInput("scm_org") || "gruntwork-io";
```

Comment:
This should be PATCHER_ORG (or PATCHER_GROUP for gitlab... probably fine to just call it org)